src/tests/MockProvider.ts

```diff
@@ -0,0 +1,88 @@
+import { readFile as _readFile } from 'fs';
+import path from 'path';
+import { promisify } from 'util';
+
+const readFile = promisify(_readFile);
+import type { APIProvider, APIResponse } from '@wharfkit/session';
+
+const __dirname = path.resolve(path.dirname(''));
+
+export interface MockInterface {
+    url: string;
+    method: 'POST' | 'GET';
+    request: {
+        contentType: string;
+        payload: any;
+    };
+    response: {
+        code: number;
+        body: string | any;
+        location: string | null;
+    };
+}
+export class MockProvider implements APIProvider {
+    async load(filename: string): Promise<MockInterface | undefined> {
+        const fullPath = __dirname + '/src/tests';
+        const filePath = path.join(fullPath, filename);
+        return await this.getMockData(filePath);
+    }
+
+    mapFilename(path: string) {
+        if (path === '/get_rows') return 'mock-data-get-rows.json';
```

Comment:
Remove the if statement if you're always returning 'mock-data-get-rows.json'

Reply:
Don't think my comment here has been addressed. This function still always returns 'mock-data-get-rows.json' so it shouldn't have the if statement.